doc/nrf/gs_installing.rst

```diff
@@ -55,6 +55,61 @@ The installation process is different depending on your operating system.
 
       Also see :ref:`zephyr:mac-setup-alts` for additional information.
 
+If you are interested in building `Connected Home over IP`_ applications, install also the `GN`_ meta-build system to generate files for `Ninja`_.
+
+.. tabs::
+
+   .. group-tab:: Windows
+
+      The Connected Home over IP project currenlty supports only Linux and macOS.
+      Support for Windows is not yet available.
+
+   .. group-tab:: Linux
+
+      To install the GN tool, complete the following steps:
+
+      1. Download the GN binary archive and extract it by using the following commands:
+
+         .. parsed-literal::
+            :class: highlight
+
+            mkdir ${HOME}/gn && cd ${HOME}/gn
+            wget -O gn.zip https://chrome-infra-packages.appspot.com/dl/gn/gn/linux-amd64/+/latest
+            unzip gn.zip gn
```

Comment:
What does the "gn" at the end do? Do you mean "-d gn" - but that would give you $HOME/gn/gn ... ?

Reply:
Yeah, right, I think I've lost `-d` somewhere and that's why I didn't notice the problem. Anyway I removed that option, as in fact `gn.zip` contains only one executable `gn` file, so I think there is not need to create another directory.